src/utils/index.js

```diff
@@ -1,21 +1,45 @@
 const MAX_VALUE_LENGTH = 5;
 
 /**
- * 콘솔 입력에 대한 유효성 검사 함수
+ * 자동차 이름들에 대한 유효성 검사 함수
  *
  * @param {string | undefined} input 입력 받은 값
  * @returns 해당 입력값이 유효한지 여부
  */
-export function isInputValid(input) {
-  const hasSplitedValues = !!input && input.split(',').length > 0;
-  const hasNonEmptyValues = input
-    .split(',')
-    .every(
-      (splitedInput) =>
-        !!splitedInput &&
-        splitedInput.trim().length > 0 &&
-        splitedInput.trim().length <= MAX_VALUE_LENGTH,
-    );
+export function isCarNamesValid(input) {
+  if (!input) return false;
 
-  return hasSplitedValues && hasNonEmptyValues;
+  const splitedValues = input.split(',');
+
+  const hasSplitedValues = splitedValues.length > 0;
+  const hasEmptyValues = splitedValues.some(
+    (splitedInput) =>
+      splitedInput.trim().length === 0 ||
+      splitedInput.trim().length > MAX_VALUE_LENGTH,
+  );
+
+  return hasSplitedValues && !hasEmptyValues;
+}
+
+/**
+ * 자동차 경주 횟수 입력에 대한 유효성 검사 함수
+ *
+ * @param {string | undefined} input 입력 받은 값
+ * @returns 해당 입력값이 유효한지 여부
+ */
+export function isLapVaild(input) {
+  if (!input) return true; // 빈 값으로 입력하면 기본 횟수로 설정
+
+  return !isNaN(Number(input));
+}
+
+/**
+ * 렌덤 정수값을 추출하는 함수
+ *
+ * @param {number} min 난수의 최소 범위의 값
+ * @param {number} max 난수의 최대 범위의 값
+ * @returns 랜덤 값
+ */
+export function getRandomNumber(min = 0, max = 10000) {
```

Comment:
소소하게 1,000 단위를 넘어가는 숫자들에는 numeric separators를 같이 사용해주시면 좋을거 같네용 ㅎㅎ

```suggestion
export function getRandomNumber(min = 0, max = 10_000) {
```